### PR TITLE
Adds prism-theme-odyssey package

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
Adds `prism-theme-odyssey` package. 

**Note:** This is a private package that is only used in `vuepress-theme-odyssey` at this time. It will not be distributed until someone requests it.

### CODE REVIEW

File: packages/prism-theme-odyssey/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        test: /\.s[ac]ss$/i,
+        use: [
+          // Creates `style` nodes from JS strings
+          'style-loader',
+          // Translates CSS into CommonJS
+          'css-loader',
+          // Compiles Sass to CSS
+          'sass-loader',
+        ],
+      },
```
Comment (arnoldsandoval-okta) [inline packages/prism-theme-odyssey/webpack.config.js:23]: > I don't quite follow where are we consuming `sass` and therefore need to use the `sass-loader`. Can you help me understand this better?
> 
>_Originally posted by @jmelberg-okta in https://github.com/okta/odyssey/pull/714#discussion_r509797918_

Comment (arnoldsandoval-okta) [reply packages/prism-theme-odyssey/webpack.config.js:23]: We are importing partials from Odyssey 

https://github.com/okta/odyssey/pull/730/files#diff-d55846964f7ead7d0690796169fb46dfdcf377eac4cdfacd28ba705a5dd1e8a8R1-R3

**Comment by edburyenegren-okta** [comment]
@alexdahl-okta not immediately relevant, but I know you've been curious about the path forward here

**Review by edburyenegren-okta — APPROVED** [review]
lgtm! css + docs context is 💯

File: packages/prism-theme-odyssey/src/index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+   <body>
+    <main>
+      <h1>Prism Theme Odyssey</h1>
+      <section>
+        <h2>HTML</h2>
+        <pre><code class="language-html">&lt;!DOCTYPE html&gt;
+  &lt;html lang="en"&gt;
+    &lt;head&gt;
+      &lt;meta charset="utf-8"&gt;
+      &lt;title&gt;Hello world&lt;/title&gt;
```
Comment (jmelberg-okta) [inline packages/prism-theme-odyssey/src/index.html:26]: Why are we HTML escaping `<`, `>`, etc?

File: packages/prism-theme-odyssey/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "devDependencies": {
+    "capture-website": "^1.1.0",
+    "html-webpack-plugin": "^4.3.0",
+    "sass": "^1.26.10",
+    "sass-loader": "^9.0.2",
+    "style-loader": "^1.2.1",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.12",
+    "webpack-dev-server": "^3.11.0"
```
Comment (jmelberg-okta) [inline packages/prism-theme-odyssey/package.json:24]: There are no `yarn.lock` file changes here - please make sure to run `yarn install` and push up those changes before merging!

**Review by jmelberg-okta — APPROVED** [review]
One question + ask. Other than that everything is fine for now!

File: packages/prism-theme-odyssey/package.json (hunk 3 above)
Comment (arnoldsandoval-okta) [reply packages/prism-theme-odyssey/package.json:24]: Thank you!

File: packages/prism-theme-odyssey/src/index.html (hunk 2 above)
Comment (arnoldsandoval-okta) [reply packages/prism-theme-odyssey/src/index.html:26]: In order to show the actual bracket within the code block. If we don't do this, the `pre` block wont render the bracket.

**Pull request merged**